backend-api/src/functions/asyncActiveSession/handlerDependencies.ts

```diff
@@ -5,37 +5,21 @@ import {
   ISessionService,
   SessionService,
 } from "../services/session/sessionService";
-import { PublicKeyGetter } from "./tokenService/publicKeyGetter";
-import {
-  ITokenService,
-  ITokenServiceDependencies,
-  TokenService,
-} from "./tokenService/tokenService";
-import { TokenVerifier } from "./tokenService/tokenVerifier";
+import { ITokenService, TokenService } from "./tokenService/tokenService";
 import { IDecryptJwe, JweDecrypter } from "./jwe/jweDecrypter";
 
-const tokenServiceDependencies: ITokenServiceDependencies = {
```

Comment:
We didn't need all these dependencies here in the end but we can have a discussion about this if people would like that.